### PR TITLE
#731 central ledger documentation

### PR DESCRIPTION
**Pull request by HenkKodde**
Included Central Ledger API specification, Database schema and diagram.

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -13,6 +13,21 @@ Documentation: [Mojaloop Specification]({{ book.importedVars.mojaloop.spec.uri.d
 | --- | --- | --- |
 | 1.0 | Initial release | 2018-11-01 |
 
+## Central Ledger API
+
+Documentation: [Central-Ledger Service](../mojaloop-technical-overview/central-ledger/README.md)
+
+* [Central Ledger API {{ book.importedVars.ledger.spec.version }} Specification](./central-ledger-api-specification.md)
+* [Swagger]({{ book.importedVars.ledger.spec.uri.api }})
```
Comment (mdebarros) [inline api/README.md]: I do not see the `book.importedVars.ledger.spec.uri.api` variable being set in the `https://github.com/mojaloop/documentation/blob/master/book_variables.yml`?

File: api/README.md [hunk 2]
```diff
@@ -13,6 +13,21 @@ Documentation: [Mojaloop Specification]({{ book.importedVars.mojaloop.spec.uri.d
...
+
+### Versions
+
+| Version | Info | Release Date |
+| --- | --- | --- |
+| 2.0 | Feature/#308 Calculate Positions implementation for Fulfil and Reject | 2018-07-30 |
```
Comment (mdebarros) [inline api/README.md]: Where did you get these versions from?

They do not seem to match the version of the swagger documentation?

I believe these need to be relooked at.

**Comment by mdebarros** [comment]
> Hi @HenkKodde. Thanks for moving and annotating documentation. I haven't reviewed your previous PR for this story, but as I'm looking at the result of this one I'm getting confused of all the info we have under **Central-Ledger Services**. I think that it could be greatly improved by utilising the capabilities of the menu. Please, consider using the following structure (may not be 100% accurate - just an example):
> 
> ```
> Central-Ledger Services
>     Transfer API
>         Transfer Prepare
>             Prepare Handler
>                 .. referenced diagrams
>             Position Handler
>                 .. referenced diagrams
>             Notifications
>                 .. references
>         Transfer Fulfil
>             Fulfil Handler
>                 .. refs
>             Fulfil Position Handler
>                 .. refs
>             Notifications
>                 .. refs
>         Reject/Abort
>             ..
>                 ..
>         Timeout
>             ..
>                 ..
>     Admin API
>         ..
> ```
> 
> I know it's a bit more work at a time when it looks the story is done and documentation is moved, but in my opinion just by creating the structure described above, we could double or even triple the readability and make central-ledger related documentation more comprehensive. Thanks.

@ggrg nice suggestion.

File: api/README.md (hunk 1 above)
Comment (HenkKodde) [reply api/README.md]: Have added the entry to the .yml file - thank you.

File: api/README.md (hunk 2 above)
Comment (HenkKodde) [reply api/README.md]: Yip - agreed. Used the change history of the .swagger file. Will chat to you to see how best the rectify this.

Comment (HenkKodde) [reply api/README.md]: Have updated as per our discussion. - thank you.

**Pull request merged**